tasks/main.yml

```diff
@@ -24,4 +24,8 @@
   when: ansible_pkg_mgr == "pacman"
 
 - import_tasks: config.yml
+  tags:
+   - ssh_config
```

Comment:
Why this tag?

Reply:
ansible-playbook allows --tags als well as --skip-tasks. Depending on the other tags used, it may be easier to use --tags="ssh_config,something_else" than to add another parameter.